<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replace_Part</class>
 <widget class="QWidget" name="Replace_Part">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="Replace_LineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Find_LineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查找内容: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>替换为: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>93</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="FindNextButton">
      <property name="text">
       <string>查找下一个</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReplaceButton">
      <property name="text">
       <string>替换</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReplaceAllButton">
      <property name="text">
       <string>全部替换</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancleButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="ReplaceIsUpper">
      <property name="text">
       <string>区分大小写</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ReplaceLoop">
      <property name="text">
       <string>循环</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
